cognite/client/_version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.34.0"
-__api_subversion__ = "V20210423"
+__version__ = "2.34.1"
+__api_subversion__ = "V20211122"
```

Comment:
api subversion doesn't need to be changed as you aren't using that afaik? it's related to header `cdf-version` as a consequence of not using playground endpoint.

Reply:
reverted 👍